templates/footer.twig

```diff
@@ -13,9 +13,7 @@
 						</a>
 					</li>
 				{% endfor %}
-			</ul>
-
-			<ul class="list-unstyled footer-links-secondary">
+				
```

Comment:
It's a bit odd that we end up with 2 different menus that just concatenated at the front end. I suppose we don't want to keep that situation indefinitely and ideally want to have people merge these to a single menu?

Reply:
I agree with that. Also take a look [this](https://jira.greenpeace.org/browse/PLANET-6189?focusedCommentId=243206&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-243206) thread.